api/src/rate-limiter.ts

```diff
@@ -14,26 +14,29 @@ type IRateLimiterOptionsOverrides = Partial<IRateLimiterOptions> | Partial<IRate
 export function createRateLimiter(configOverrides?: IRateLimiterOptionsOverrides): RateLimiterAbstract {
 	switch (env.RATE_LIMITER_STORE) {
 		case 'redis':
-			return new RateLimiterRedis(getConfig('redis', configOverrides));
+			return new RateLimiterRedis(getRedisConfig(configOverrides));
 		case 'memory':
 		default:
-			return new RateLimiterMemory(getConfig('memory', configOverrides));
+			return new RateLimiterMemory(getMemoryConfig(configOverrides));
 	}
 }
 
-function getConfig(store: 'memory', overrides?: IRateLimiterOptionsOverrides): IRateLimiterOptions;
-function getConfig(store: 'redis', overrides?: IRateLimiterOptionsOverrides): IRateLimiterStoreOptions;
-function getConfig(
-	store: 'memory' | 'redis' = 'memory',
-	overrides?: IRateLimiterOptionsOverrides
-): IRateLimiterOptions | IRateLimiterStoreOptions {
-	const config: any = getConfigFromEnv('RATE_LIMITER_', `RATE_LIMITER_${store}_`);
-
-	if (store === 'redis') {
-		const Redis = require('ioredis');
-		delete config.redis;
-		config.storeClient = new Redis(env.RATE_LIMITER_REDIS || getConfigFromEnv('RATE_LIMITER_REDIS_'));
-	}
+function getMemoryConfig(overrides?: IRateLimiterOptionsOverrides): IRateLimiterOptions {
+	const config: any = getConfigFromEnv('RATE_LIMITER_', `RATE_LIMITER_MEMORY_`);
```

Comment:
Do we need to omit `RATE_LIMITER_MEMORY_`? I think it isn't expected that config takes place through this variable, at least it's not documented.

Reply:
Same question as @paescuj otherwise, everything looks good.